src/west/app/project.py

```diff
@@ -932,6 +940,10 @@ def update(self, project):
                 stats['checkout new manifest-rev'] = perf_counter() - start
             _post_checkout_help(project, current_branch, sha, is_ancestor)
 
+        # Check whether project has any submodules and if it does update them.
+        if project.submodules:
```

Comment:
The function checks if an argument is null, perhaps you can skip the check here?

Reply:
:+1:

Reply:
Done